addons/l10n_ro_edi/models/account_move.py

```diff
@@ -12,14 +12,18 @@ class AccountMove(models.Model):
     )
     l10n_ro_edi_state = fields.Selection(
         selection=[
-            ('invoice_sending', 'Sending'),
-            ('invoice_sent', 'Sent'),
+            ('invoice_sending', 'Sent'),
+            ('invoice_sent', 'Validated'),
         ],
         string='E-Factura Status',
         compute='_compute_l10n_ro_edi_state',
         store=True,
+        help="""- Sent: Successfully sent to the SPV, waiting for validation
```

Comment:
Because the selection key values is hidden from the user, but actually I think I'll change that to make it less confusing :) 

The move's state helper was written more for the document's state, and the documents have and can show the error state.

Reply:
Ok, let me know when I can check the review again :)